development-apache/docker-httpd.conf

```diff
@@ -555,23 +555,25 @@ ServerTokens Prod
 EnableSendfile on
 
 <VirtualHost *:443>
-    ProxyPass / http://janus_app:3000/
+    ProxyPass / http://janus_app:3000/ disablereuse=on
     ProxyPassReverse / http://janus_app:3000/
 
     ServerName janus.development.local
 
     SSLEngine on
     SSLCertificateFile /root/certs/janus.development.local.pem
     SSLCertificateKeyFile /root/certs/janus.development.local-key.pem
+    Header add Service-Worker-Allowed "/"
```

Comment:
These have been reflected already in cbc-chef repo as well.  Necessary for service workers to be allowed on any path of the metis program.  If we do not include this header, we would be forced to serve the downzip-sw.js file in the `/` directory instead of the `/js` directory because service workers by default are not allowed to control pages that are not children of their own paths.  Check documentation on Service-Worker-Allowed and service worker scopes for more info.